src/decoder/lattice-faster-decoder.h

```diff
@@ -95,32 +102,31 @@ namespace decoder {
 // versions of the decoder that support quickly getting the best path
 // (LatticeFasterOnlineDecoder, see lattice-faster-online-decoder.h) and also
 // those that do not (LatticeFasterDecoder).
-
-
-// ForwardLinks are the links from a token to a token on the next frame.
-// or sometimes on the current frame (for input-epsilon links).
-template <typename Token>
-struct ForwardLink {
-  using Label = fst::StdArc::Label;
-
+template<typename Token>
+struct Link {
   Token *next_tok;  // the next token [or NULL if represents final-state]
-  Label ilabel;  // ilabel on arc
-  Label olabel;  // olabel on arc
-  BaseFloat graph_cost;  // graph cost of traversing arc (contains LM, etc.)
-  BaseFloat acoustic_cost;  // acoustic cost (pre-scaled) of traversing arc
-  ForwardLink *next;  // next in singly-linked list of forward arcs (arcs
-                      // in the state-level lattice) from a token.
-  inline ForwardLink(Token *next_tok, Label ilabel, Label olabel,
-                     BaseFloat graph_cost, BaseFloat acoustic_cost,
-                     ForwardLink *next):
-      next_tok(next_tok), ilabel(ilabel), olabel(olabel),
-      graph_cost(graph_cost), acoustic_cost(acoustic_cost),
-      next(next) { }
+  fst::StdArc::Label i_label;
```

Comment:
If there is no compelling reason to change the names to i_label and o_label, I would prefer the original names because they mirror the names used in OpenFst.

Reply:
Got it.